src/main/java/pl/task/parkingmeter/controller/VehicleController.java

```diff
@@ -0,0 +1,190 @@
+package pl.task.parkingmeter.controller;
+
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.web.bind.annotation.*;
+import org.springframework.web.servlet.config.annotation.EnableWebMvc;
+import pl.task.parkingmeter.entity.Profit;
+import pl.task.parkingmeter.entity.Rates;
+import pl.task.parkingmeter.entity.Vehicle;
+import pl.task.parkingmeter.exception.InvalidDateException;
+import pl.task.parkingmeter.exception.InvalidRegNumberException;
+import pl.task.parkingmeter.exception.VehicleAddedEarlierException;
+import pl.task.parkingmeter.exception.VehicleNotFoundException;
+import pl.task.parkingmeter.repository.RatesService;
+import pl.task.parkingmeter.repository.ValueRateService;
+import pl.task.parkingmeter.repository.VehicleService;
+
+import java.math.BigDecimal;
+import java.sql.Timestamp;
+import java.time.LocalDateTime;
+import java.time.temporal.ChronoUnit;
+import java.util.ArrayList;
+import java.util.List;
+import java.util.regex.Matcher;
+import java.util.regex.Pattern;
+
+@RestController
+@RequestMapping("/")
+@EnableWebMvc
```

Comment:
czy ta adnotacja nie powinna być raczej na jakiejś klasiekonfiguracyjnej?
oczywiście tu też zadziała, ale to kwestia porządku

Reply:
bez adnotacji tutaj lub w klasie testowej, część testów nie przechodziła mi pomyślnie